chrome-launcher/README.md

```diff
@@ -111,9 +111,10 @@ chromeLauncher.launch({
 
 ### Continuous Integration
 
-In a CI environment like Travis, Chrome may not be installed. If you want to use `chrome-launcher`, you can install Chrome using Lighthouse's `download-chrome.sh` script:
+In a CI environment, Chrome may not be installed. If you want to use `chrome-launcher`, e.g. Travis can [install Chrome at run time with an addon](https://docs.travis-ci.com/user/chrome).
```

Comment:
@brendankenny I changed your message a bit... wdyt?

Reply:
After rereading that, I realized that your one is better here, so I pushed new commit https://github.com/GoogleChrome/lighthouse/pull/3377/commits/4cb66353fede765e11ee45bbae8b6a85a44b9d60. Thanks :)